test/spec/features/ordering/BpmnOrderingProviderSpec.js

```diff
@@ -343,6 +343,48 @@ describe('features/modeling - ordering', function() {
 
     });
 
+
+    describe('inside subprocess', function() {
+
+      var diagramXML = require('./collapsed-subprocess.bpmn');
+
+      beforeEach(bootstrapModeler(diagramXML, { modules: testModules }));
+
+
+      describe('should stay always in front', function() {
+
+        it('moving <Group> onto <StartEvent>', inject(function() {
+
+          // when
+          move('Group', { x: 100, y: 0 }, 'StartEvent', false);
```

Comment:
To consider: Add an integration test to ensure groups can be added in _collapsed_ sub-processes.

Reply:
Added with [39dd523](https://github.com/bpmn-io/bpmn-js/pull/1599/commits/39dd5235614fef390c73c9dc2f624d2ad3b402bd)